core/statistics/resourceMonitor.go

```diff
@@ -34,14 +35,34 @@ func (rm *ResourceMonitor) GenerateStatistics() string {
 	var memStats runtime.MemStats
 	runtime.ReadMemStats(&memStats)
 
-	return fmt.Sprintf("timestamp: %d, uptime: %v, num go: %d, go mem: %s, sys mem: %s, total mem: %s, num GC: %d\n",
+	fileDescriptors := int32(0)
+	numOpenFiles := 0
+	numConns := 0
+	proc, err := getCurrentProcess()
+	if err == nil {
+		fileDescriptors, _ = proc.NumFDs()
+		openFiles, err := proc.OpenFiles()
+		if err == nil {
+			numOpenFiles = len(openFiles)
```

Comment:
Is the number of file descriptors currently in use different from the number of open files?

Reply:
No, they differ. At least on linux.